imageledger/jinja2/results.html

```diff
@@ -8,7 +8,11 @@
 
 {% with %}
   {% set endpoint = '' %}
-  {% include "includes/search-form.html" %}
+  {% if results.items|length == 0 or request.GET.get('advanced', '') != '' %}
```

Comment:
Thing here is that results.items|length returns 0 also when results (or results.items) is undefined, which is the case in the first screen. Calling length () saves a little space.